test/BF16/Integration/mlp-all.mlir

```diff
@@ -1,10 +1,10 @@
-// RUN: tpp-opt %s -pack-matmul="block-factors=32,32,32" -pack-vnni  -generalize-tensor-pack-unpack -bufferize -rewrite-to-brgemm -convert-vnni-to-tpp -canonicalize -convert-tpp-to-xsmm -convert-xsmm-to-func -canonicalize -convert-check-to-loops -convert-linalg-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine -convert-arith-to-llvm -convert-vector-to-llvm -finalize-memref-to-llvm -arith-expand -convert-math-to-llvm -convert-func-to-llvm -reconcile-unrealized-casts |\
+// RUN: tpp-opt %s -pack-matmul="block-factors=32,32,32" -pack-vnni  -generalize-tensor-pack-unpack -bufferize -rewrite-to-brgemm -canonicalize -convert-tpp-to-xsmm -convert-xsmm-to-func -canonicalize -convert-check-to-loops -convert-linalg-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine -convert-arith-to-llvm -convert-vector-to-llvm -finalize-memref-to-llvm -arith-expand -convert-math-to-llvm -convert-func-to-llvm -reconcile-unrealized-casts |\
```

Comment:
This needs to change to `tpp-opt --default-tpp-passes` instead and run with `tpp-run`.